mirrord/agent/src/steal/ip_tables/redirect.rs

```diff
@@ -98,6 +98,83 @@ where
     }
 }
 
+pub struct StandardRedirect<IPT: IPTables> {
+    preroute: PreroutingRedirect<IPT>,
```

Comment:
I see some usage of preroute's `managed` field and this can cause problems with the unloading nature of the ip table guards, I would recommend using a separate `IPTableChain` for this Redirect
```suggestion
    preroute: PreroutingRedirect<IPT>,
    managed: IPTableChain<IPT>,
```